BACKEND/userAuth/src/main/java/com/apptime/auth/repository/TaskRepository.java

```diff
@@ -24,7 +24,6 @@
 
 	Task deleteById(long id);
 
-	 
-	 
-
+	@Query("select u from Task u where u.scheduledstart > :start and u.userName = :name")
+    Set<Task> getSpecificDayTasks( Date start,String name);
```

Comment:
Shall we rename the name as `getTasksStartedLaterThan..` or something else? The current name could not tell users what it would do.